README.md

```diff
@@ -73,7 +91,22 @@ As you can imagine each of these apps are are deployed by ArgoCD to the Kind clu
 
 [Excalidraw diagram here:](https://excalidraw.com/#json=MNOQf_OeLtKYe_Y80Bt2l,AP-ftLAwZoDWjp2yudnMKA)
 
-
+## To start developing
```

Comment:
I think we can push this section up. Just below the quick start section. I would also rename this section to "Developer notes"